docs/guides/private-submodules.rst

```diff
@@ -71,48 +52,21 @@ to give read access to several repositories using only one SSH key.
    #. Click on :guilabel:`SSH and GPG keys`
    #. Click on :guilabel:`New SSH key`
    #. Put a descriptive title and paste the
-   #. :ref:`public SSH key from your Read the Docs project <guides/private-submodules:Copy your project's SSH key>`
+      :ref:`public SSH key from your Read the Docs project <guides/importing-private-repositories:copy your project's public key>`
    #. Click on :guilabel:`Add SSH key`
 
-GitLab
-~~~~~~
+Azure DevOps
+------------
 
-For GitLab, Read the Docs uses `deploy keys with read only access <https://docs.gitlab.com/ee/ssh/#deploy-keys>`__,
-which allows you to reuse a SSH key across different repositories.
-Since Read the Docs already added the public SSH key on your main repository,
-you only need to add it to each repository of your submodules.
-
-#. Go to the project of your submodule on GitLab
-#. Click on :guilabel:`Settings`
-#. Click on :guilabel:`Repository`
-#. Expand the :guilabel:`Deploy Keys` section
-#. Put a descriptive title and paste the
-#. :ref:`public SSH key from your Read the Docs project <guides/private-submodules:Copy your project's SSH key>`
-#. Click on :guilabel:`Add key`
-#. Repeat the previous steps for each submodule
-
-Bitbucket
-~~~~~~~~~
-
-For Bitbucket, Read the Docs uses `access keys with read only access <https://confluence.atlassian.com/bitbucket/access-keys-294486051.html>`__,
-which allows you to reuse a SSH key across different repositories.
-Since Read the Docs already set the public SSH key on your main repository,
-you only need to add it to each repository of your submodules.
-
-#. Go to the project of your submodule on Bitbucket
-#. Click on :guilabel:`Settings`
-#. Click on :guilabel:`Access keys`
-#. Click on :guilabel:`Add key`
-#. Put a descriptive label and paste the
-#. :ref:`public SSH key from your Read the Docs project <guides/private-submodules:Copy your project's SSH key>`
-#. Click on :guilabel:`Add key`
-#. Repeat the previous steps for each submodule
+Azure DevOps doesn't have per-project keys.
```

Comment:
You could probably either drop this section, as it's not providing any particular advice, or maybe expand the explanation here a bit more. Missing bits of information from this section are:

* SSH keys are instead attached to users
* As long as the user has access to your repository and all submodule repositories, RTD can clone all the repos with the same key

Reply:
Changed the content to include your suggestions :+1: